### DISCUSSION TITLE
The livewire:navigated event fires before livewire:initialized on page load

### DISCUSSION THREAD

**Discussion by joelwmale** [livewire/livewire#7373] [7. Bugs (archived)] [answered]
### Livewire version

3.2.1

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

Add an event listener for `livewire:navigated` and then listen to a Livewire event using `Livewire.on(....)`

You'll see the following event on page load (or something similar):

```
Uncaught ReferenceError: Livewire is not defined
```

If you don't refresh the page, and navigate instead, you get no errors because Livewire is already technically loaded.

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joelwmale** [comment]
The issue is definitely just the registration of the navigation plugin occurs on line 27 of `lifecycle.js` before Livewire/Alpine are fully initialised, which results in `Livewire` being undefined

**Comment by joshhanley** [comment, marked as answer]
We're in the process of investigating all wire:navigate issues and I have tested out your scenario. I believe this has been fixed so I'm going to close this. If you're still experiencing problems, please open a new discussion. Thanks!
